assets/pages/showcase/showcase.scss

```diff
@@ -0,0 +1,213 @@
+// -----  gu-sass ----- //
+
+@import '~stylesheets/gu-sass/gu-sass';
+
+// ----- Shared Components ----- //
+@import '~components/footer/footer';
+@import '~components/headers/simpleHeader/simpleHeader';
+
+@import '~components/svgs/svg';
+
+@import '~components/headingBlock/headingBlock';
+@import '~components/leftMarginSection/leftMarginSection';
+@import '~components/productPage/productPageHero/productPageHero';
+@import '~components/productPage/productPageContentBlock/productPageContentBlock';
+@import '~components/productPage/productPageTextBlock/productPageTextBlock';
+@import '~components/productPage/productPageButton/productPageButton';
+@import '~components/productPage/productPageFeatures/productPageFeatures';
+
+:root {
+  scroll-behavior: smooth;
+}
+
+#showcase-landing-page {
+  .component-product-page-hero {
+    @include mq($from: tablet) {
+      .component-heading-block {
+        margin-top: 24em;
```

Comment:
Is there a reason for using the relative `em` unit here? Would `px` work instead? They get converted to `rem` in the build.

Reply:
This was just stolen from https://github.com/guardian/support-frontend/blob/eb3eba67be7317cf558fc1fb85191a9665facae7/assets/pages/paper-subscription-landing/paperSubscriptionLandingPage.scss#L29. We should DRY this out in a subsequent PR.